purview/logger.py

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+import logging
+import sys
+
+
+# Inspired from waiverdb
+def log_to_stdout(level=logging.INFO):
+    fmt = '%(asctime)s [pid %(process)5d] %(name)s %(levelname)s %(message)s'
```

Comment:
I don't think we care about the pid (`[pid %(process)5d]`) in the log statements. It's just extra noise.